public/app/plugins/datasource/cloudwatch/components/LogsQueryField.test.tsx

```diff
@@ -6,12 +6,40 @@ import { DescribeLogGroupsRequest } from '../types';
 import { SelectableValue } from '@grafana/data';
 // eslint-disable-next-line lodash/import-scope
 import _, { DebouncedFunc } from 'lodash';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
 
 jest
   .spyOn(_, 'debounce')
   .mockImplementation((func: (...args: any) => any, wait?: number) => func as DebouncedFunc<typeof func>);
 
 describe('CloudWatchLogsQueryField', () => {
+  it('runs onRunQuery on blur of Log Groups', async () => {
+    const onRunQuery = jest.fn();
+
+    render(
+      <CloudWatchLogsQueryField
+        absoluteRange={{ from: 1, to: 10 }}
+        exploreId={ExploreId.left}
+        datasource={
+          {
+            getVariables: () => [],
+            getRegions: () => Promise.resolve([]),
+          } as any
```

Comment:
I believe we have some sort of function that's like setupMockDatasource or something like that, might be nice to use that right?